macros/models/_sector/dex/platforms/uniswap_v2_forked_base_trades.sql

```diff
@@ -0,0 +1,51 @@
+{% macro uniswap_v2_forked_base_trades(
+    blockchain = null
+    , project = null
+    , version = null
+    , Pair_evt_Swap = null
+    , Factory_evt_PairCreated = null
+    )
+%}
+
+WITH dexs AS
+(
+    SELECT
+        t.evt_block_number AS block_number
+        ,t.evt_block_time AS block_time
+        ,t.to AS taker
+        ,t.contract_address AS maker
```

Comment:
Also this line we may need to discuss, which is better for default `maker`? 
`t.contract_address` or `CAST(NULL AS VARBINARY)`?

Reply:
for now, let's leave as-is, since this is how it was in existing dex trades